bbp/nse/projectionizer/default.nix

```diff
@@ -1,72 +1,69 @@
-{ stdenv
+{ bluepy
+, brainbuilder
 , config
 , fetchgitPrivate
+, flatindexer
 , pythonPackages
+, stdenv
 , voxcell
-, flatindexer
-, bluepy
-, brainbuilder
 }:
 
 
 
 let
 in
 pythonPackages.buildPythonPackage rec {
-	version = "1.1.1";
+  version = "1.1.1";
   name = "projectionizer-${version}";
 
-   src = fetchgitPrivate {
-       url = "ssh://bbpcode.epfl.ch/building/Projectionizer";
-       rev = "1f85d69c122bbf684ac249982eb2e09eae94b0e2";
-       sha256 = "1j6bsyqs5zrj9lh1d1cdz34g1br5sid03zw8r7j5v9349si6kpic";
-   };
+  src = fetchgitPrivate {
+    url = "ssh://bbpcode.epfl.ch/building/Projectionizer";
+    rev = "1f85d69c122bbf684ac249982eb2e09eae94b0e2";
+    sha256 = "1j6bsyqs5zrj9lh1d1cdz34g1br5sid03zw8r7j5v9349si6kpic";
+  };
 
-    preConfigure = ''
-        # use >= deps and not absolute == to avoid conflicts....
-        sed -i 's/==\([^ ]\)/>=\1/g' setup.py
-    '';
+  preConfigure = ''
+    # use >= deps and not absolute == to avoid conflicts....
+    sed -i 's/==\([^ ]\)/>=\1/g' setup.py
+  '';
 
   buildInputs = with pythonPackages; [
-      nose
-      mock
+    mock
+    nose
   ];
 
-	propagatedBuildInputs = with pythonPackages; [
-							  setuptools
-							  enum34
-							  future
-							  scipy
-							  numpy
-							  pyyaml
-							  tqdm
-							  matplotlib
-							  pyarrow
-							  partd
-							  luigi
-                pandas
-                distributed
-							] ++ [
-                voxcell
-                bluepy
-                brainbuilder
-                flatindexer
-              ];
-
-    passthru = {
-        pythonDeps = stdenv.propagatedBuildInputs;
-    };
-
-    # TODO: Remove this fix when https://github.com/BlueBrain/bbp-nixpkgs/issues/952
-    # is fixed
-    postPatch = ''
-            echo "Removing libFLATIndex dependency for python >= 3.4"
-            sed -i -e "s/['\"]libFLATIndex.*$//" setup.py
-    '';
+  propagatedBuildInputs = with pythonPackages; [
+    bluepy
+    brainbuilder
+    distributed
+    enum34
+    flatindexer
+    future
+    luigi
+    matplotlib
+    numpy
+    pandas
+    partd
+    pyarrow
+    pyyaml
+    scipy
+    setuptools
+    tqdm
+    voxcell
+  ];
 
-    checkPhase = ''
-        nosetests
-    '';
+  passthru = {
```

Comment:
What does it do ?

Reply:
`passthru` allows you to access package properties outside the derivation.
https://nixos.org/nixpkgs/manual/

Here it means that anywhere you can access package propagatedBuildInputs through `projectionizer.pythonDeps` property.